src/frontend/src/lib/components/ui/RoundedIcon.svelte

```diff
@@ -2,11 +2,11 @@
 	import type { ComponentType } from 'svelte';
 
 	export let icon: ComponentType;
-	export let iconStyleClass = '';
+	export let withOpacity = false;
```

Comment:
We can also call the property just "opacity"

Reply:
changed